src/lib/AlertWindow.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import ModalWindow from "./ModalWindow";
+import Button from "./Button";
+
+const InfoText = styled.div`
+  text-align: center;
+`;
+
+const ButtonRow = styled.div`
+  text-align: center;
+  margin-top: 32px;
+  
+  button {
+    margin: 0 16px;
+  }
+`;
+
+const AlertWindow = (props) => (
+  <ModalWindow {...props}>
+    <InfoText>{props.info}</InfoText>
+    <ButtonRow>
+      {props.buttons.map((button, idx) =>
+        <Button key={button.name + idx} rounded size="medium" onClick={button.onClick} theme={button.theme}>
+          {button.name}
+        </Button>
+      )}
+    </ButtonRow>
+  </ModalWindow>
+);
+
+AlertWindow.propTypes = {
+  height: PropTypes.string,
+  width: PropTypes.string,
+  title: PropTypes.string,
+  info: PropTypes.string,
+  buttons: PropTypes.arrayOf(PropTypes.object),
```

Comment:
1. Are any of these required? Then you can also use:
`PropTypes.string.isRequired`

2. Could you also use PropTypes.shape instead of PropTypes.object?

Reply:
Done